File: Assistant/lib/view/core_task_worker/setting.dart
import '/common.dart';
import '/utility/storage_path.dart';

// ----------------

class Setting {
  StoragePath       kernel;
  StoragePath       script;
  List<String>      argument;
  Boolean           immediateLaunch;
  List<StoragePath> messageFont;
  Setting({
    required this.kernel,
    required this.script,
    required this.argument,
    required this.immediateLaunch,
    required this.messageFont,
  });
}

File: AssistantPlus/Application/View/CoreTaskWorker/Setting.cs
#pragma warning disable 0,
// ReSharper disable

using Twinning.AssistantPlus;
using Twinning.AssistantPlus.Utility;

namespace Twinning.AssistantPlus.View.CoreTaskWorker {

	public record Setting {
		public StoragePath  Kernel          { get; set; } = default!;
		public StoragePath  Script          { get; set; } = default!;
		public List<String> Argument        { get; set; } = default!;
		public Boolean      AutomaticScroll { get; set; } = default!;
		public Boolean      ImmediateLaunch { get; set; } = default!;
		public String       MessageFont     { get; set; } = default!;
	}

}
